app/importer/server/classes/ImportDataConverter.ts

```diff
@@ -235,7 +242,13 @@ export class ImportDataConverter {
 		this.addUserServices(updateData, userData);
 		this.addUserImportId(updateData, userData);
 		this.addUserEmails(updateData, userData, existingUser.emails || []);
-		Users.update({ _id }, updateData);
+
+		if (Object.keys(updateData.$set).length === 0) {
+			delete updateData.$set;
+		}
+		if (Object.keys(updateData).length > 0) {
+			Users.update({ _id }, updateData);
+		}
```

Comment:
I think changing it to a single statement makes it easier to read, wdyt? 
```suggestion
		if (Object.keys(updateData.$set).length > 0) {
			Users.update({ _id }, updateData);
		}
```

Reply:
No, `updateData` can have an `$addToSet` too.

Reply:
ok.. I didn't realize the calls could mutate `updateData` 👍